lib/csv_to_popolo.rb

```diff
@@ -339,30 +353,72 @@ def given?(key)
       @r.key?(key) && !@r[key].nil? && !@r[key].empty?
     end
 
+    def split_cell(key)
+      separator = MODEL[key][:multivalue_separator]
+      if separator
+        @r[key].split(separator)
+      else
+        [@r[key]]
+      end
+    end
+
+    def set_cell_from_values(key, values)
```

Comment:
This was already being done in two places:
* https://github.com/tmtmtmtm/csv_to_popolo/blob/d8850022875aac7914a94e67ac1750643be3c6ee/lib/csv_to_popolo.rb#L343-L346
* https://github.com/tmtmtmtm/csv_to_popolo/blob/d8850022875aac7914a94e67ac1750643be3c6ee/lib/csv_to_popolo.rb#L358-L360

... so I assumed you were OK with that. I can change that if you like, though.